src/Chainweb/Cut.hs

```diff
@@ -240,6 +242,35 @@ unsafeMkCut v hdrs = Cut'
     , _cutChainwebVersion = v
     }
 
+-- In some scenarios it's useful to reset a node's latest cut to some
+-- past cut, which we can specify either as the ancestor of the latest
+-- cut at some block height or as the cut sitting in a file at the indicated path.
+data CutResetTarget
```

Comment:
Where is this used? Are we planning to use this in a future iteration of the reset logic?

Maybe we should come up with a concrete list of scenarios for cut initialization and reset logic.

Reply:
It depends if we plan to use that in the future. I think, the ability to reset to a particular cut could be useful to resolve forks or also for testing. But I am not sure.

Reply:
Deleted for now.